### PR TITLE
Initial commit to set community responsibilities

### PR DESCRIPTION
**Pull request by mazzystr**
Initial commit to set community responsibilities definition

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,18 +1,17 @@
 reviewers:
-  - codificat
```
Comment (maya-r) [inline OWNERS:2]: stray diff? this feels like it doesn't match the content of the rest of the PR

Comment (mazzystr) [reply OWNERS:2]: You're right.  I can remove this diff.

File: responsibilities/community_communications.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Community communications
```
Comment (dhiller) [inline responsibilities/community_communications.md:1]: I don't see any link from the responsibilities README to this document, shouldn't it have a link?

File: responsibilities/community_communications.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  * \#virtualization
+  * \#kubevirt-dev
+
+* Twitter
+
+  * KubeVirt
+
+## CNCF Code of Conduct
+
+If you're here then now is a good time ot review the [CNCF Code of conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
```
Comment (dhiller) [inline responsibilities/community_communications.md:22]:
```suggestion
If you're here then now is a good time to review the [CNCF Code of conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
```

File: responsibilities/community_events.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+* 45 min + demonstration
+  * TBD
+* 1 hour + demonstration
+  * TBD
+
+## Events
+
+### Red Hat Summit / https://www.redhat.com/en/summit
+
+Red Hat is curerntly the main sponsor of project KubeVirt.  It is expected that KubeVirt
```
Comment (dhiller) [inline responsibilities/community_events.md:37]:
```suggestion
Red Hat is currently the main sponsor of project KubeVirt.  It is expected that KubeVirt
```

File: responsibilities/community_meeting.md [hunk 5]
```diff
@@ -20,48 +22,58 @@ At KubeVirt's present scale the community opts to host a weekly meetings for all
...
 
 * Recordings
-  * Community organizers will process Zoom mp4 digital audio/video recording into a recording using Open Source codecs [VP9](https://en.wikipedia.org/wiki/VP9) video and [Opus](https://en.wikipedia.org/wiki/Opus_%28audio_format%29) audio
-```
-  cd zoom/DATE
-  ffmpeg -i "${I}" -c:v libvpx-vp9 -c:a libopus $(basename "`pwd`" | awk {'print $1 "_" $3 "_" $4 "_" $5'}).webm
-```
-  * Community organizers will upload the video to KubeVirt’s YouTube channel and the Community Meetings playlist.
+  * The Zoom meeting is set to record to the platform
+  * Recording needs to be dowloaded then uploaded to KubeVirt’s YouTube channel
```
Comment (dhiller) [inline responsibilities/community_meeting.md:26]:
```suggestion
  * Recording needs to be downloaded then uploaded to KubeVirt’s YouTube channel
```

**Review by dhiller — CHANGES_REQUESTED** [review]
Some nitpicking only...

**Comment by dhiller** [comment]
One general comment: I think it helps the user if a README links to the subtopics that are covered. Is this by intent that some subtopics are linked and others aren't?

**Comment by dhiller** [comment]
Regarding roadmap: I found this seemingly very outdated wiki on kubevirt/kubevirt: https://github.com/kubevirt/kubevirt/wiki/ Should this be deleted and moved into some other form or do we want to keep this, and update the content?

File: responsibilities/community_communications.md (hunk 2 above)
Comment (mazzystr) [reply responsibilities/community_communications.md:1]: Link added

File: responsibilities/community_meeting.md (hunk 5 above)
Comment (mazzystr) [reply responsibilities/community_meeting.md]: It's almost like we need spell check on the community repo.... ;)

Fixed

**Comment by mazzystr** [comment]
> One general comment: I think it helps the user if a README links to the subtopics that are covered. Is this by intent that some subtopics are linked and others aren't?

This is an excellent topic for the community meeting

**Pull request closed without merging**